network/modules/UDP/client_socket.py

```diff
@@ -0,0 +1,108 @@
+import socket
+from network.modules.UDP.socket_wrapper import UdpSocket
+
+
+class UdpClientSocket(UdpSocket):
+    """
+    Wrapper for client socket operations
+    """
+
+    __create_key = object()
+
+    def __init__(
+        self,
+        class_private_create_key: object,
+        socket_instance: socket.socket,
+        server_address: tuple,
+    ) -> None:
+        """
+        Private Constructor, use create() method.
+        """
+        assert class_private_create_key is UdpClientSocket.__create_key
+        self.__socket = socket_instance
+        self.server_address = server_address
+
+    @classmethod
+    def create(
```

Comment:
Can we add a parameter for setting a timeout? Have a default of like 10seconds (see `settimeout()`)